frontend/Main.elm

```diff
@@ -575,7 +575,7 @@ logoImage : Int -> H.Html msg
 logoImage width =
     H.img
         [ A.alt "Tradenomiitti"
-        , A.src "/static/tradenomiitti_logo.svg"
+        , A.src "/static/main_logo.svg"
```

Comment:
For the images that have been in the repo before: what if we kept them in `conf/example-assets/`?